apps/portals/eliteportal/src/config/synapseConfigs/handleFilesToParticipants.tsx

```diff
@@ -4,10 +4,10 @@ import {
 } from '@sage-bionetworks/synapse-types'
 import {
   SynapseClient,
-  SynapseConstants,
   SynapseUtilityFunctions,
```

Comment:
Just a note: all of the non-type imports from 'synapse-react-client' trigger importing the entire bundle. So just separating SynapseConstants doesn't change anything when you still have this.

No need to fix in this PR, we can eventually do a separate pass once all of the portals have been migrated to this pattern.